pkg/nvml/zz_generated.api.go

```diff
@@ -668,6 +667,7 @@ type Interface interface {
 	VgpuTypeGetName(VgpuTypeId) (string, Return)
 	VgpuTypeGetNumDisplayHeads(VgpuTypeId) (int, Return)
 	VgpuTypeGetResolution(VgpuTypeId, int) (uint32, uint32, Return)
+	Extensions() ExtendedInterface
```

Comment:
Why / how did this get generated at the bottom? The generation code sorts the functions alphabetically ...